### ISSUE TITLE
Good script, but there might be a better way

### ISSUE THREAD

**Issue by homebysix** [TedRinehart/AdwareMedic-SH#1]
I came across your script. Useful!

Have you used [AutoPkg](https://github.com/autopkg/autopkg)? Once up and running, it would enable you to produce an up-to-date .pkg installer for AdwareMedic in a matter of seconds. That .pkg could be deployed using ARD, the same way you'd deploy your script.

If you're just getting started with AutoPkg, you might check out the GUI app called [AutoPkgr](https://github.com/lindegroup/autopkgr) my team made that makes setup easy.

Anyway, thought I'd pass that along. Hope it saves you time someday!

**Comment by TedRinehart** [comment]
Thanks for checking out the repo; I appreciate the comment.

I'm familiar with AutoPkg, but chose to take this method for other reasons. Part of the reason I pushed a script like this to GitHub was to keep things simple. While the README indicates that it was designed with ARD in mind, it also has excellent potential for being applied when out in the field without a utility drive containing the DMG, or in environments that mass deployment is not feasible/necessary. If you're just servicing a machine or two, or providing in-home support to a client, deployment isn't something you're concerned with.

In such a scenario a tech could open the user's browser, probably bogged down with adware, fight a few redirects and eventually download AdwareMedic themselves and clean the machine. Alternately, they could just open up the Terminal and blast off the curl command to run this script, and be ready to go with much less hassle. This is why I chose to work with a script rather than a package, for the flexibility in the event of a "what if" scenario, or one in which deployment is out of the question.

While there's definitely other ways to accomplish the same/a similar result, whether or not one is better than the other depends on the situation. For my purposes and those who I intended to use this, having that extra flexibility means that this repo housing a script that can be directly called remotely is the best way.
